### PR TITLE
Bump sandbox to use CentOS 7.4 boxes

### PR DESCRIPTION
**Pull request by jgrund**

### CODE REVIEW

File: hpc-storage-sandbox-el7/Vagrantfile [hunk 1]
```diff
@@ -21,20 +21,20 @@ Vagrant.configure("2") do |config|
 
 	# Hostname prefix for the cluster nodes
 	# Example conventions:
-	# ct<vmax><vmin>: CentOS <vmax>.<vmin>, e.g. ct73 = CentOS 7.3
-	# rh<vmax><vmin>: RHEL <vmax>.<vmin>, e.g. rh73 = RHEL 7.3
+	# ct<vmax><vmin>: CentOS <vmax>.<vmin>, e.g. ct74 = CentOS 7.4
```
Comment (brianjmurrell) [inline hpc-storage-sandbox-el7/Vagrantfile]: Does it really matter to have the version in the names of the boxes?

Any scripts that leverage this file will need to be constantly updated also.

Comment (jgrund) [reply hpc-storage-sandbox-el7/Vagrantfile]: Those are host_names for the boxes, not the box name itself.

Also, there aren't any derivative scripts that use this file.

Comment (jgrund) [reply hpc-storage-sandbox-el7/Vagrantfile]: I updated the comments to reflect that this file only uses the major version.

Comment (brianjmurrell) [reply hpc-storage-sandbox-el7/Vagrantfile]: > Those are host_names for the boxes, not the box name itself.

Right.  But regardless, some script writer leveraging this file will need to go update all of their scripts.

> Also, there aren't any derivative scripts that use this file.

That you are aware of/that we [do not currently] have.  You don't know what scripts other people out there using this file have.  We should try to keep the update burden for such users to a minimum.

If having the version number in the host names serves a useful purpose, that is fine.  I'm just not seeing what that purpose is at the moment.

**Pull request merged**